packages/cli/src/constructs/http-check.ts

```diff
@@ -0,0 +1,58 @@
+import { Check, CheckProps } from './check'
+import { HttpRequest } from './http-request'
+import { Session } from './project'
+
+export interface HttpCheckProps extends Omit<CheckProps, 'doubleCheck' | 'shouldFail'> {
```

Comment:
everything related to scripting needs to be omitted as well: tearDownScript and so on

Reply:
How about environment variables? I guess they will still be usable in headers and such?